files/install-requirements.yaml

```diff
@@ -8,13 +8,17 @@
       - make
       - git
       - dnf-utils
+      - gcc
```

Comment:
why do we need gcc, krb5-devel and python3-devel?

Reply:
This was caused by dependencies from `rebase-helper`, especially `dnf -y builddep packit.spec` in `files/install-requirements.yaml`. But after a week. All is working properly.
I removed.